components/chat/ToolCallApproval.tsx

```
@@ -0,0 +1,101 @@
+  toolCall,
+  isOpen,
+  onApprove,
+  onReject,
+}: ToolCallApprovalProps) {
+  // Format the tool call arguments for display
+  const formatArgs = (args: Record<string, any>) => {
+    return JSON.stringify(args, null, 2);
+  };
+
+  // Get a human-readable description of the tool call
+  const getToolDescription = (name: string, args: Record<string, any>) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)